src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsId;
+    private Long userId;
```

Comment:
checklist: When choosing the fields to be returned in your ShoppingCartResponseDto, keep in mind that cartId and userId hold the same value.